languages/julia/exercises/concept/annalyns-infiltration2/.docs/instructions.md

```diff
@@ -0,0 +1,100 @@
+In this exercise, you'll be extending the quest logic for the new RPG game a friend is developing.
+To make the game more interesting, your friend wants you to add random events that change the situation.
+
+The general setting hasn't changed from the previous exercise.
+The new random events are highlighted in the text below.
+
+The game's main character is Annalyn, a brave girl with a fierce and loyal pet dog.
+Unfortunately, disaster strikes, as her best friend was kidnapped while searching for berries in the forest.
+Annalyn will try to find and free her best friend, optionally taking her dog with her on this quest.
+
+After some time spent following her best friend's trail, she finds the camp in which her best friend is imprisoned. It turns out there are two kidnappers: a mighty knight and a cunning archer.
+
+Having found the kidnappers, Annalyn considers which of the following actions she can engage in:
+
+- _Fast attack_: a fast attack can be made if the knight is sleeping, as it takes time for him to get his armor on, so he will be vulnerable.<!-- **There is a 5% chance that Annalyn stumbles while approaching the knight. This will wake up the knight and buy him enough time to put on his armor so that a fast attack is no longer possible.** -->
```

Comment:
Is the comment bit there because you toyed with adding that as an additional rule? Other maintainers might wonder why that bit is commented.

Reply:
Yes, that's why. I'll add an explanation to the comment

Reply:
👍